### ISSUE TITLE
'WebDriverException: Message: Illegal base64 character a' trying to send a file path to a file upload input using RemoteDriver

### ISSUE THREAD

**Issue by scrofts** [SeleniumHQ/selenium#2906]
## Meta -
OS:  OSX 10.10.5
Selenium Version: selenium server 3.0.0-beta4 and python bindings 3.0.0b3
Browser:  Firefox and Chrome
Browser Version: Firefox 49.0.1 and Chrome 53.0.2785.143 (64-bit)

## Expected Behavior -
File path should be sent to file upload input.

## Actual Behavior -
Exception is thrown:
*** WebDriverException: Message: Illegal base64 character a
Stacktrace:
    at java.util.Base64$Decoder.decode0 (Base64.java:714)
    at java.util.Base64$Decoder.decode (Base64.java:526)
    at java.util.Base64$Decoder.decode (Base64.java:549)
    at org.openqa.selenium.io.Zip.unzip (Zip.java:120)
    at org.openqa.selenium.remote.server.handler.UploadFile.call (UploadFile.java:42)
    at org.openqa.selenium.remote.server.handler.UploadFile.call (UploadFile.java:29)
    at java.util.concurrent.FutureTask.run (FutureTask.java:266)
    at org.openqa.selenium.remote.server.DefaultSession$1.run (DefaultSession.java:176)
    at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1142)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:617)
    at java.lang.Thread.run (Thread.java:745)

## Steps to reproduce -
from selenium.webdriver.remote.webdriver import WebDriver as RemoteDriver
from selenium.webdriver.common.desired_capabilities import DesiredCapabilities

file_to_upload = "/file/that/exists/on/your/machine"

try:
    driver = RemoteDriver(command_executor="http://127.0.0.1:4444/wd/hub", desired_capabilities=getattr(DesiredCapabilities, "CHROME"))
    driver.get("https://jsfiddle.net/sqod9632/1/")
    driver.switch_to.frame("result")
    driver.find_element_by_id("file").send_keys(file_to_upload)
finally:
    driver.quit()

**Comment by scrofts** [comment]
Only an issue for me when using RemoteDriver.

**Comment by shs96c** [comment]
When using the Java bindings and the latest build from master, this works as expected. Possibly a problem with the way that python is base64 encoding things?


```
RemoteWebDriver driver =
      new RemoteWebDriver(new URL("http://localhost:4444/wd/hub"), DesiredCapabilities.chrome());
    driver.setFileDetector(new LocalFileDetector());

    driver.get("https://jsfiddle.net/sqod9632/1/");
    driver.switchTo().frame("result");
    driver.findElement(By.id("file")).sendKeys("/path/to/hello.txt");

```

**Comment by AutomatedTester** [comment]
@scrofts what version of Python are you using?

**Comment by scrofts** [comment]
I'm using Python 2.7.3

**Comment by lukeis** [comment]
was fixed again by simon ;) 10f20c70054f9b5ceb4215b7d63578f29343b722
